content/modules/mobile-travel-tutorial/pages/swift/develop/security.adoc

```diff
@@ -10,7 +10,7 @@
 User is created via the Travel sample web app.
 When a user is created, a corresponding user profile document is created on Couchbase Server associated with the user.
 In addition, the web app automatically registers the user with the Sync Gateway via the Sync Gateway
-xref:sync-gateway:rest-api-admin.adoc#/user/post__db___user_[admin RestAPI] endpoint.
+xref:sync-gateway::rest-api-admin.adoc[admin RestAPI] endpoint.
```

Comment:
You've lost the fragment. Looks like we'd want to link to https://docs.couchbase.com/sync-gateway/current/rest-api-admin.html#/Database_Security/post_db__user_ (e.g. `user` has been renamed to `Database_Security`)

Alternatively could link to the static version https://docs.couchbase.com/sync-gateway/current/rest_api_admin_static.html#post_db-_user-

Reply:
Fixed. (I think)